src/lib/SendbirdState.tsx

```diff
@@ -3,8 +3,9 @@ import type { User } from '@sendbird/chat';
 import type { GroupChannel, Member } from '@sendbird/chat/groupChannel';
 import { Dispatch } from 'react';
 import { MarkAsReadSchedulerType } from './hooks/useMarkAsReadScheduler';
+import { ChannelActionTypes } from '../smart-components/Channel/context/dux/actionTypes';
 
-export type CustomUseReducerDispatcher = Dispatch<{ type: string, payload: any }>;
+export type CustomUseReducerDispatcher = Dispatch<{ type: ChannelActionTypes, payload: any }>;
```

Comment:
This makes all the actionTypes into: `RESET_MESSAGES" | "FETCH_INITIAL_MESSAGES_START" | "FETCH_INITIAL_MESSAGES_SUCCESS" | "FETCH_INITIAL_MESSAGES_FAILURE" | "FETCH_PREV_MESSAGES_SUCCESS" | "FETCH_PREV_MESSAGES_FAILURE" etc...`  becuase this is a patch, just call it string.. We can refactor reducers using the pattern from https://github.com/sendbird/sendbird-uikit-react/pull/486